man/systemd-journal-gatewayd.service.xml

```diff
@@ -277,9 +277,13 @@
     <para>
       <option>Range: entries=<replaceable>cursor</replaceable>[[:<replaceable>num_skip</replaceable>]:<replaceable>num_entries</replaceable>]</option>
     </para>
+    <para>
+      <option>Range: time=[<replaceable>since</replaceable>]:[<replaceable>until</replaceable>][[:<replaceable>num_skip</replaceable>]:<replaceable>num_entries</replaceable>]</option>
+    </para>
 
     <para>where
       <replaceable>cursor</replaceable> is a cursor string,
+      <replaceable>since</replaceable> and <replaceable>until</replaceable> are timestamps,
```

Comment:
say explicit in which unit and whith which base and which clock

Reply:
-> I've put it like in the `date` manual page ("(seconds since 1970-01-01 00:00:00 UTC)").